tests/pass-dep/libc/prctl.rs

```diff
@@ -0,0 +1,56 @@
+//@ignore-target: windows # No prctl on Windows
+//@ignore-target: apple # No prctl on macOS
```

Comment:
`@only-target: linux android # prctl is Linux-specific` would probably be better

Reply:
Since I'm moving the code to in `android/foreign_items.rs` the description should be like `Supported for Android only" or something like that.